_rules/presentational-children-no-focusable-content-307n5z.md

```diff
@@ -0,0 +1,142 @@
+---
+id: 307n5z
+name: Element with presentational children has no focusable content
+rule_type: atomic
+description: |
+  This rule checks that elements with a role that makes it children presentational do not contain focusable elements.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:4.1.2: # Name, Role, Value (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+The rule applies to any HTML or SVG element with a [semantic role][] that defines its [children][child] to be [presentational children][].
+
+## Expectation
+
+None of the target elements have [descendants][] in the [flat tree][] that are part of [sequential focus navigation][].
+
+## Assumptions
+
+_There are currently no assumptions_
+
+## Accessibility Support
+
+Several major browsers ignore the WAI-ARIA requirements on [presentational children][] for most or sometimes all roles. Because some browsers do, and others do not implement presentational children, there can be big differences between browsers.
+
+## Background
+
+Elements with a [semantic role][] that has [presentational children][] will not have any descendants in the accessibility tree. If any of those descendants are included in [sequential focus navigation][], this causes the focus to land on an element that has no corresponding node in the [accessibility tree][]. The result is that there is no programmatic name or role available for assistive technologies. There are other problems that can come from [presentational children][] too. These must be tested separately.
+
```

Comment:
Wondering whether we should include a list of roles with presentational children here (like we do for some of the form field rules targeting "role that inherit from input/select and have no required context that inherit from input/select".

Reply:
Maybe, lets leave this open, see what other reviewers think.